TestApp/android/build.gradle

```diff
@@ -27,3 +27,15 @@ allprojects {
         }
     }
 }
+
+subprojects {
```

Comment:
Do we still need this workaround with the other fix?

Reply:
@jwallra Yes we still need this for external dependency `react-native-fs` and `react-native-image-picker` in TestApp.

`react-native-fs` is on `buildToolsVersion "25.0.0"` and TestApp is on `buildToolsVersion "27.0.3"`, so without this workaround Android Studio will give the following warning:

```
WARNING: The specified Android SDK Build Tools version (25.0.0) is ignored, as it is below the minimum supported version (27.0.3) for Android Gradle Plugin 3.1.3.
```